pipeline/conf.py

```diff
@@ -78,11 +78,11 @@
     'LESS_ARGUMENTS': '',
 
     'MIMETYPES': (
-        (b'text/coffeescript', '.coffee'),
-        (b'text/less', '.less'),
-        (b'text/javascript', '.js'),
-        (b'text/x-sass', '.sass'),
-        (b'text/x-scss', '.scss')
+        (str('text/coffeescript'), str('.coffee')),
+        (str('text/less'), str('.less')),
+        (str('text/javascript'), str('.js')),
+        (str('text/x-sass'), str('.sass')),
+        (str('text/x-scss'), str('.scss'))
```

Comment:
These would be prettier if we could leave them as bare literals, but because the module has `from __future__ import unicode_literals` we have to call `str()` on them.